src/main/java/vazkii/botania/common/item/relic/ItemLokiRing.java

```diff
@@ -151,9 +151,13 @@ public static void renderHUDNotification(){
 		GTNHLib.proxy.printMessageAboveHotbar(text, 60, true, true);
 	}
 	public static String getLokiModeText(ItemStack stack){
-		return EnumChatFormatting.GOLD + StatCollector.translateToLocal("item.botania:lokiRing.name") + " " + (isRingEnabled(stack) ?
-				EnumChatFormatting.GREEN + StatCollector.translateToLocal("botaniamisc.lokiOn") :
-				EnumChatFormatting.RED + StatCollector.translateToLocal("botaniamisc.lokiOff"));
+		String text = EnumChatFormatting.GOLD + StatCollector.translateToLocal("item.botania:lokiRing.name") + " ";
+		if (isRingEnabled(stack)) {
+			text = text + EnumChatFormatting.GREEN + StatCollector.translateToLocal("botaniamisc.lokiOn");
+		} else {
+			text = text + EnumChatFormatting.RED + StatCollector.translateToLocal("botaniamisc.lokiOff");
+		}
+		return text;
```

Comment:
This can be reverted since it doesn't change anything

Reply:
yeah it was mostly so that I could debug from where the problem came. I can revert it now tho. Just gonna copy paste the old one